src/SignalR/server/Core/src/Internal/DefaultHubDispatcher.cs

```diff
@@ -73,7 +73,7 @@ internal partial class DefaultHubDispatcher<THub> : HubDispatcher<THub> where TH
 
         public override async Task OnConnectedAsync(HubConnectionContext connection)
         {
-            var scope = _serviceScopeFactory.CreateScope();
+            await using var scope = _serviceScopeFactory.CreateAsyncScope();
```

Comment:
either remove `await using` or remove the `try finally` below

Reply:
remove the try finally.

Reply:
I'll let you and @davidfowl hash this out, he suggested adding await using for these in a previous suggestion